src/shared/components/DashboardEditor/DashboardConfigEditor.tsx

```diff
@@ -14,38 +16,35 @@ export type DashboardPayload = {
   plugins?: string[];
 };
 
-type Plugin = {
-  key: string;
-  title: string;
-  description?: string;
-  chosen: boolean;
-};
-
 export type DashboardConfigEditorProps = {
   ref?: React.Ref<FormComponentProps<any>>;
   form: WrappedFormUtils;
   dashboard?: DashboardPayload;
-  availablePlugins?: Plugin[];
   onSubmit?(dashboard: DashboardPayload): void;
   viewList?: ResourceList<{}>;
   linkToSparqlQueryEditor?(dataQuery: string): React.ReactElement;
 };
 
 const DashboardConfigEditorComponent: React.FunctionComponent<
   DashboardConfigEditorProps
-> = ({
-  onSubmit,
-  form,
-  dashboard,
-  availablePlugins,
-  linkToSparqlQueryEditor,
-}) => {
+> = ({ onSubmit, form, dashboard, linkToSparqlQueryEditor }) => {
+  const avaliablePlugins =
+    useSelector((state: RootState) => state.config.plugins) || [];
```

Comment:
maybe move this state logic to a container

Reply:
👍